src/sentry/api/endpoints/sentry_app_installation_external_issue_details.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework.response import Response
+
+from sentry.api.bases import (
+    SentryAppInstallationExternalIssueBaseEndpoint as ExternalIssueBaseEndpoint,
+)
+from sentry.mediators import external_issues
+from sentry.models import PlatformExternalIssue
+
+
+class SentryAppInstallationExternalIssueDetailsEndpoint(ExternalIssueBaseEndpoint):
+    def delete(self, request, installation, external_issue_id):
+        try:
+            platform_external_issue = PlatformExternalIssue.objects.get(
```

Comment:
@ceorourke What is preventing someone from picking an external issue from another org and using it here? We need to check the organization linked through the project on `PlatformExternalIssue` to make sure it's the same org as the installation.